lib/recurly/billing_info.php

```diff
@@ -1,5 +1,10 @@
 <?php
-
+/*
+ * class Recurly_BillingInfo
+ * @property string $account_code Account's unique code.
+ * @property string $token_id A token generated by Recurly.js
+ * @property string $currency Currency in which invoices will be posted. Only applicable if this account is enrolled in a plan has a different currency than your site's default.
```

Comment:
There's definitely more properties on this object: https://dev.recurly.com/docs/create-an-accounts-billing-info-credit-card